app/src/main/java/com/example/todoapp/presentation/navigation/Transition.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.todoapp.presentation.navigation
+
+import androidx.compose.animation.AnimatedContentTransitionScope
+import androidx.compose.animation.EnterTransition
+import androidx.compose.animation.ExitTransition
+import androidx.compose.animation.core.Spring
+import androidx.compose.animation.core.spring
+import androidx.navigation.NavBackStackEntry
+
+/**
+ * Shows transition animation after navigating to screen.
+ */
+fun AnimatedContentTransitionScope<NavBackStackEntry>.enterTransition(): EnterTransition {
+    return slideIntoContainer(
+        towards = AnimatedContentTransitionScope.SlideDirection.Left,
+        animationSpec = spring(stiffness = Spring.StiffnessMediumLow),
+    )
+}
+
+/**
+ * Shows transition animation after navigating from screen.
```

Comment:
nit: I don't mind your use of english in this project comments, but it will be more convenient if you will stick to one language (because in the beginning you have comments in Russian and in my opnion some sort of mix language documentations may be not that stylish)